lib/v2/deltaio/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'delta.io': {
+        _name: 'Delta Lake',
+        '.': [
+            {
+                title: 'Delta Lake',
+                docs: 'https://docs.rsshub.app/blog#deltalake',
```

Comment:
```suggestion
                title: 'Blogs',
                docs: 'https://docs.rsshub.app/routes/blog#deltalake',
```

Reply:
In the example of this [doc,](https://docs.rsshub.app/zh/joinus/new-radar) the docs attribute is  https://docs.rsshub.app/programming#github, without routes.
I think this doc needs to be updated.